mappings/net/minecraft/text/EntityNbtComponent.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS jr net/minecraft/text/NbtTextComponent
+CLASS jr net/minecraft/text/EntityNbtComponent
```

Comment:
This class should simply be called `NbtComponent`, `EntityNbtComponent` is the inner class (`jr$b`).